libs/gisday/platform/ngx/data-access/src/lib/modules/services/signage/signage.service.ts

```diff
@@ -1,21 +1,30 @@
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 
+import { OidcSecurityService } from 'angular-auth-oidc-client';
+
 @Injectable({
   providedIn: 'root'
 })
 export class SignageService {
-  public withCredentials = false;
+  private accessToken;
+  private headers: HttpHeaders;
 
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, public oidcSecurityService: OidcSecurityService) {
+    this.accessToken = this.oidcSecurityService.getAccessToken();
```

Comment:
Duplication of logic. Another reason to go with the http interceptor in the future.